automation_testing/tests/FormFields/formFieldColorPicker.spec.ts

```diff
@@ -0,0 +1,41 @@
+import { test, expect } from "@playwright/test";
+import { FormFieldColorPickerPage } from "../../pages/FormFields/FormFieldColorPickerPage";
+import { getRandomHexCode } from "../../utils/helpers/helper";
+
+test.describe("FormFields - FormFieldImageUpload - Kitchen Sink", () => {
```

Comment:
This says FormFieldImageUpload, if it's the only issue then I'll change it directly here.